TestingApp/test/fetch.spec.js

```diff
@@ -1546,7 +1424,7 @@ describe("Fetch AuthHttpRequest class tests", function() {
             const data = await response.json();
             assertEqual(data.message, "test");
 
-            assert.equal(refreshCalled, 1);
+            assert.equal(refreshCalled, 0);
```

Comment:
So we changed the mobile sdk logic to not make a refresh call if the front token is missing on app launch (The web SDKs do this to consider cases where sessions may exist because Safari clears periodically). This test needed to change to account for one less refresh call